### DISCUSSION TITLE
Multiple "filelists" per mod ( for enabling/disabling 'optional' content & functions )

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15162] [Ideas and suggestions]
As is, mods only have one filelist which includes all of it's content loaded at once.

Since alot of needless duplicates of mods come from stuff like tweaked stat variants, edited standalone content from a larger mod, compatibility patches with other mods, and other sorts of compartmentalized content expansions & options.

Thus my feature idea is allowing one 'mod package' to define multiple file lists with different names to appear in the in-game mod manager, for example:
- one filelist could be the the complete mod's content, or the "core" of the mod.
- another would be some kind if 'lite' version which loads different content
- or an optional expansion on top of the "core" that adds content some people could choose to play with or without
- or even a built-in compatibility patch with another mod, which loads the necessary overwrites when enabled.

The idea is somewhat inspired by what modding in some games like Rimworld or Project Zomboid does, in which mods can have multiple content packs...

**Comment by jigardi** [comment]
Have my upvote.
This would make mods a lot easier to maintain, and to develop.
